doc/onboarding/view/step3.md

```diff
@@ -0,0 +1,106 @@
+
+
+## Step 3/5 : Create a password
+
+### URI
+
+`:userID/password`
+
+
+### Models
+
+<pre>
+user = {
+	&lt;id&gt; id,
+}
+</pre>
+<pre>
+form = {
+	&lt;boolean&gt; disabled: true
+	fields: [
+		{
+		    &lt;slug&gt; name: 'password',
+		    &lt;string&gt; label: 'Password'
+		    &lt;string&gt; type: 'password'
+		    &lt;password&gt; value
+		}
+	]
+}
+</pre>
+<pre>
+step = {
+	&lt;string&gt; slug: 'set_password',
+	&lt;step&gt; value: 3/5,
+}
+</pre>
+
+
+### Getters
+
+<pre>
+    getPasswordComplexity: (state) ->
+        // Apply algorithm to know
+        // if password value is secured enough
+        // Add a value >= 0 and =< 1
+        // return { complexity, value }
+
+
+    getNextStepURI: (state) ->
+        // Route getter should "know" the relation
+        // between screens
+        // check state values
+        // return <URI> URI
+</pre>
+
+
+### Actions
+
+<pre>
+    getFormData: ->
+        return { password: @state.password, id: @state.userID }
+
+
+    validate: ->
+        { password } = @getFormData()
```

Comment:
Onboarding, no, but Step, I think yes. For me a step is aware of how to validate itself. Maybe my example was bad : the onboarding should not know that a given step needs explicit `password` to validate itself. Maybe the call should look like:
```javascript
@step.validate()
// or
@step.validate(@step.getData())
```
Was it your point ?